### DISCUSSION TITLE
Timelines - > Count Only First Occurrence option apply to each of the added concepts?

### DISCUSSION THREAD

**Discussion by lawdank** [uwrit/leaf#594] [Q&A] [answered]
Hi Leaf Folks,

Could you confirm whether **Count Only First Occurrence** option applies to each of the added concepts for Timelines?

Thanks!

**Comment by ndobb** [comment, marked as answer]
Hi @lawdank - yes, **Count Only First Occurrence** applies to each of the added concepts on Timelines. If you see something off or strange, please let us know and we can take a look.

**Reply by lawdank** [reply]
Hi @ndobb - Thank you for confirming. We are still testing our data, will keep an eye out for any anomalies
